README.md

```diff
@@ -17,8 +17,8 @@ For more information on using Helm, refer to the [Helm's documentation](https://
 
 Just do the following:
 ```
-$ helm repo add teracyhq-incubator https://teracyhq-incubator.storage.googleapis.com/
-$ helm install teracyhq-incubator/<chart-name>
+$ helm repo add teracyhq-incubator https://teracyhq_incubator.storage.googleapis.com/
+$ helm install teracyhq_incubator/<chart-name>
```

Comment:
teracyhq-incubator

Reply:
temporary use this util we move to official domain